.github/workflows/ci.yml

```diff
@@ -325,9 +325,9 @@ jobs:
           # TODO: would like to adopt `actionlint` pre-commit hook
           # but false positive here and inability to do an inline ignore
           # prevents this https://github.com/rhysd/actionlint/issues/237
-          - python-version: ${{ github.event_name == 'pull_request' && '3.9' }}
-          - python-version: ${{ github.event_name == 'pull_request' && '3.10' }}
-          - python-version: ${{ github.event_name == 'pull_request' && '3.11' }}
+          - python-version: ${{ (github.event_name == 'pull_request' && '3.9') || (github.event == 'merge_group' && '3.9') }}
+          - python-version: ${{ (github.event_name == 'pull_request' && '3.10') || (github.event == 'merge_group' && '3.10') }}
+          - python-version: ${{ (github.event_name == 'pull_request' && '3.11') || (github.event == 'merge_group' && '3.11') }}
```

Comment:
I'm sure there's a more succinct way to do this, but everything else I've tried fails.